inference-engine/src/vpu/common/src/ngraph/query_network.cpp

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "vpu/ngraph/query_network.hpp"
+#include "ngraph/op/util/op_types.hpp"
+#include "ngraph/opsets/opset3.hpp"
+
+#include <transformations/rt_info/fused_names_attribute.hpp>
+#include <legacy/details/ie_cnn_network_iterator.hpp>
+#include <ie_algorithm.hpp>
+
+
+namespace vpu {
+
+InferenceEngine::QueryNetworkResult getQueryNetwork(const InferenceEngine::ICNNNetwork::Ptr& convertedNetwork,
+                                                    const std::shared_ptr<const ngraph::Function>& function,
+                                                    const std::string& pluginName, const std::set<std::string>& supportedLayers) {
+    InferenceEngine::QueryNetworkResult res;
+    std::unordered_set<std::string> originalOps;
+    for (auto& node : function->get_ops()) {
+        originalOps.emplace(node->get_friendly_name());
+    }
+
+    std::unordered_set<std::string> supported;
+    std::unordered_set<std::string> unsupported;
+
+    std::unordered_set<std::string> splitNames;
+    std::unordered_set<std::string> concatNames;
+
+    ngraph::NodeVector splits;
+    ngraph::NodeVector concats;
+
+    const auto isLayerSupported = [&supportedLayers, &splitNames, &concatNames, &concats, &splits]
+                                  (InferenceEngine::details::CNNNetworkIterator& layer) -> bool {
+            auto node = (*layer)->getNode();
+            if (std::dynamic_pointer_cast<const ::ngraph::opset3::Split>(node) != nullptr) {
+                splitNames.emplace(node->get_friendly_name());
+                splits.push_back(node);
+                return false;
+            } else if (std::dynamic_pointer_cast<const ::ngraph::opset3::Concat>(node) != nullptr) {
+                concatNames.emplace(node->get_friendly_name());
+                concats.push_back(node);
+                return false;
+            } else {
+                return supportedLayers.count((*layer)->name) != 0;
+            }
+    };
+
+    for (InferenceEngine::details::CNNNetworkIterator itLayer{convertedNetwork.get()};
+             itLayer != InferenceEngine::details::CNNNetworkIterator();
+             itLayer++) {
+            const auto fusedNode = (*itLayer)->getNode();
```

Comment:
Some problems with indents

Reply:
The line `const auto fusedNode = (*itLayer)->getNode();` should have indents as 4 spaces, not 8